dds_web/api/api_s3_connector.py

```diff
@@ -87,10 +87,12 @@ def remove_bucket(self, *args, **kwargs):
     @bucket_must_exists
     def remove_multiple(self, items, *args, **kwargs):
         """Removes all with prefix."""
-        _ = self.resource.meta.client.delete_objects(
-            Bucket=self.project.bucket,
-            Delete={"Objects": [{"Key": x} for x in items]},
-        )
+        # s3 can only delete 1000 objects per request
+        for i in range(0, len(items), 1000):
+            _ = self.resource.meta.client.delete_objects(
+                Bucket=self.project.bucket,
+                Delete={"Objects": [{"Key": x} for x in items[i : i + 1000]]},
+            )
```

Comment:
```suggestion
        # s3 can only delete 1000 objects per request   
        for i in range(0, len(items), batch_size):
            _ = self.resource.meta.client.delete_objects(
                Bucket=self.project.bucket,
                Delete={"Objects": [{"Key": x} for x in items[i : i + batch_size]]},
            )
```
And a check to make sure the batch size is 1000 at most?